File: c/exam/q2.m
numberCourse = input("How many courses do you want to enrol in? ");
course = strings(numberCourse);
check = 0;
for index = 1:numberCourse
    fprintf("course %d", index);
    word = input(" ", 's');
    course(index) = word;
    if(course(index) == "ENG 1002" || course(index) == "ENG 1003")
        check = 1;
    end
end
 
if(check == 1)
    fprintf("Awesome! You are enrolled in programming.");
else
    fprintf("Awwww, no programming for you.");
end

File: c/exam/q2.c
# include<stdio.h>
# include<stdlib.h>
#include<string.h>

int main(void){
    int numberCourses = 0;
    int check =0;
    printf("How many courses do you want to enrol in? ");
    scanf("%d", &numberCourses);
    scanf("%*c");
    char ** course = malloc(sizeof(char*)*numberCourses);
    for (int i =0; i < numberCourses; i++){
        char * word = malloc(sizeof(char)*10);
        printf("course %d: ", i);
        scanf("%[^'\n']", word);
        scanf("%*c");
        course[i] = word;
        printf("%s", word);
        if(strcmp(word,"ENG 1002") == 0||strcmp(word,"ENG 1003") == 0){
            check = 1;
        }
    }
    if(check == 1){
        printf("Awesome! You are enrolled in programming. \n");
    }
    else{
        printf("Awwww, no programming for you. \n");
    }
}
